packages/varlet-ui/src/picker/docs/en-US.md

```diff
@@ -71,6 +71,149 @@ const picker = async () => {
 </template>
 ```
 
+### Text Formatter
+
+Picker passes in a `textFormatter` attribute to customize the text.
+`textFormatter` accepts two parameters. The first parameter `text` is the current text, and the second parameter  `columnIndex` is the subscript of the column where the current text is located.
+
+```html
+<script setup>
+import { Picker } from '@varlet/ui'
+
+const months = genCounts(12)
+
+const leapYearFebruaryDates = genCounts(29)
+
+const februaryDates = genCounts(28)
+
+const oddMonthDates = genCounts(31)
+
+const evenMonthDates = genCounts(30)
+
+function isOddMonth(month) {
+  return [1, 3, 5, 7, 8, 10, 12].includes(month)
+}
+
+function genCounts(length) {
+  return Array.from({ length }, (_, index) => index + 1)
+}
+
+function isLeapYear(year) {
+  return (year % 4 === 0 && year % 100 !== 0) || year % 400 === 0
+}
+
+function genDates(year, month) {
+  if (isLeapYear(year) && month === 2) {
```

Comment:
I think is `February` more important than `isLeapYear`. Maybe we should to judge whether February isLeapYear first. If it's not `February`,This matter is meaningless.`